CHANGELOG.md

```diff
@@ -45,6 +45,8 @@ for further information about branching and tagging conventions.
 
 #### Removed
 - "Fee" column from the movements table
+- Sale statistics in all `sale.record`
+- Displaying quantity investors in sails details
```

Comment:
* "quantity of investors" or "investors quantity"
* sails details -> sale details

Reply:
ok